06_correlation_analysis.ipynb

```diff
@@ -0,0 +1,1984 @@
+{
```

Comment:
<p>So columns here are cancer subtypes and you have a column for all cancers (<code>pancan</code>).  Just making sure I'm following.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pancancer-evaluation/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Yep!</p>